src/main/scala/com/malliina/push/apns/confs.scala

```diff
@@ -32,6 +34,26 @@ object APNSTokenConf extends ConfHelper[APNSTokenConf] {
       file <- read("private_key")
       teamId <- read("team_id")
       keyId <- read("key_id")
-    } yield APNSTokenConf(Paths get file, KeyId(keyId), TeamId(teamId))
+    } yield APNSTokenConf(Source.fromFile(file), KeyId(keyId), TeamId(teamId))
   }
+
+  def apply(
+    privateKey: Path,
+    keyId: KeyId,
+    teamId: TeamId
+  ): APNSTokenConf = new APNSTokenConf(
+    Source.fromFile(privateKey.toFile),
+    keyId,
+    teamId
+  )
+
+  def apply(
```

Comment:
Since `APNSTokenConf` is a `case class` with the same parameters as in this `apply` method, it looks like this factory method is redundant since it is automatically generated by the compiler in any case.